lsp/source/server.mts

```diff
@@ -288,7 +294,7 @@ connection.onCompletion(async ({ textDocument, position, context: _context }) =>
   // Don't map for files that don't have a sourcemap (plain .ts for example)
   if (sourcemapLines) {
     position = forwardMap(sourcemapLines, position)
-    console.log('remapped')
+    // console.log('remapped')
```

Comment:
It's fine to delete these rather than comment them out. Or log them to connection, possibly gated by a `--verbose` setting if feeling ambitious.

Reply:
Yes, what you said is correct. I do want to add a `--verbose` parameter. The work here is just relatively simple brute-force commenting.

In the past two days, I mainly want to make the unit tests run. In the next work, I will improve what you said and what I am going to do.